File: CS/HyperLinkPropertyEditor.Web/Module.cs
using System;
using DevExpress.ExpressApp;
using System.Collections.Generic;
using DevExpress.ExpressApp.Updating;

namespace HyperLinkPropertyEditor.Web {
	public sealed partial class HyperLinkPropertyEditorAspNetModule : ModuleBase {
		public HyperLinkPropertyEditorAspNetModule() {
			InitializeComponent();
		}
		protected override IEnumerable<Type> GetRegularTypes() {
			return new Type[] { typeof(WebHyperLinkStringPropertyEditor) };
		}
		protected override IEnumerable<Type> GetDeclaredExportedTypes() {
			return Type.EmptyTypes;
		}
		protected override IEnumerable<Type> GetDeclaredControllerTypes() {
			return Type.EmptyTypes;
		}
		public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
			return ModuleUpdater.EmptyModuleUpdaters;
		}
	}
}

File: VB/HyperLinkPropertyEditor.Web/Module.vb
Imports System
Imports DevExpress.ExpressApp
Imports System.Collections.Generic
Imports DevExpress.ExpressApp.Updating

Namespace HyperLinkPropertyEditor.Web
    Public NotInheritable Partial Class HyperLinkPropertyEditorAspNetModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides Function GetRegularTypes() As IEnumerable(Of Type)
            Return New Type() { GetType(WebHyperLinkStringPropertyEditor) }
        End Function
        Protected Overrides Function GetDeclaredExportedTypes() As IEnumerable(Of Type)
            Return Type.EmptyTypes
        End Function
        Protected Overrides Function GetDeclaredControllerTypes() As IEnumerable(Of Type)
            Return Type.EmptyTypes
        End Function
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
            Return ModuleUpdater.EmptyModuleUpdaters
        End Function
    End Class
End Namespace
